mlir/lib/Dialect/MIOpen/Transforms/AlignTiling.cpp

```diff
@@ -378,6 +378,10 @@ template <typename T> struct MILARewritePattern : public OpRewritePattern<T> {
     // generic and a copy operation or transform.
     int inpUses = 0;
     for (Operation *use : inp.getUsers()) {
+      if (dyn_cast<memref::DeallocOp>(use)) {
```

Comment:
`isa<>` over a `dyn_cast<>` that's never used ... come to think it, I'm surprised there's not a warning about that